test/lib/clients/inst_productsources_test.rb

```diff
@@ -7,11 +7,22 @@
 describe Yast::InstProductsourcesClient do
   subject(:client) { described_class.new }
 
+  let(:product_version) { "15.1" }
+
   describe "#main" do
     before do
       allow(Yast::Sequencer).to receive(:Run)
       allow(Yast::Wizard).to receive(:OpenDialog)
       allow(Yast::Wizard).to receive(:CloseDialog)
+      allow(Yast::Product).to receive(:version).and_return(product_version)
+    end
+
+    around do |test|
```

Comment:
Do we still need this "around" block when we mock `ENV[]=` ?

Reply:
yes, as other tests run also main, which sets ENV

Reply:
As we discussed in IRC, we will keep the `around` hook because 

```
<jreidinger> [...] all other examples also run client.main
<jreidinger> dgdavid: so it modifies ENV of whole test and can have consequences
```